paper-fab.html

```diff
@@ -103,12 +108,14 @@
       iron-icon {
         @apply(--paper-fab-iron-icon);
       }
-      
+
       span {
         width: 100%;
         white-space: nowrap;
         overflow: hidden;
         text-overflow: ellipsis;
+        text-align: center;
+        @apply(--paper-fab-label);
```

Comment:
TBH, I added the mixin because I was going to use an emoji in the demo, and emoji aren't centered (the glyph has a whack right padding), like this: '😻'. So if you wanted to add an emoji, it would always look off center and you'd have to manually compensate for it (and the mixin lets you do that)